web/themes/custom/mytheme/src/scss/region1.scss

```diff
@@ -1,24 +1,77 @@
-// News slider
+@import "variables";
+@import "mixin";
```

Comment:
no need to include "variables" and "mixin" each time, they are included in the "Global.scss" already, just include it instead.